numpy/core/src/umath/override.c

```diff
@@ -460,31 +210,26 @@ normalize_at_args(PyUFuncObject *ufunc, PyObject *args,
  */
 NPY_NO_EXPORT int
 PyUFunc_CheckOverride(PyUFuncObject *ufunc, char *method,
-                      PyObject *args, PyObject *kwds,
-                      PyObject **result)
+        PyObject *in_args, PyObject *out_args,
+        PyObject *const *args, Py_ssize_t len_args, PyObject *kwnames,
+        PyObject **result)
 {
-    int i;
-    int j;
     int status;
 
     int num_override_args;
     PyObject *with_override[NPY_MAXARGS];
     PyObject *array_ufunc_methods[NPY_MAXARGS];
 
-    PyObject *out;
-
     PyObject *method_name = NULL;
-    PyObject *normal_args = NULL; /* normal_* holds normalized arguments. */
     PyObject *normal_kwds = NULL;
 
     PyObject *override_args = NULL;
-    Py_ssize_t len;
 
     /*
      * Check inputs for overrides
      */
     num_override_args = get_array_ufunc_overrides(
```

Comment:
This change means that one can no longer override on `where` - I think this is OK, but just to be sure.

Reply:
Hmmm, that would be a bigger change then I realized.  But I think it is effectively unchanged, because by NumPy rules `where` has to be passed as a keyword argument, and we ever checked `out` when it comes to kwargs?

Reply:
Oh, technically you could pass `where` by position to `ufunc.reduce` and get dispatching.  That seems to border on insanity though :) (for one you also have to pass `initial` and everything else.)

Will write a release note now and include this.

Reply:
Indeed, I don't think this will ever have been done in the wild!